src/components/work/ContactHostFormModal/ContactHostFormModal.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import { Formik, Field, Form as FormikForm } from 'formik';
+import * as Yup from 'yup';
+import { compose, graphql } from 'react-apollo';
+
+import { Booking } from 'networking/bookings';
+import { CONTACT_USER, ContactUserField, User } from 'networking/users';
+import { Button, Form, FormGroup, Label, FormFeedback, Input, ModalFooter, ModalBody, Modal, ModalHeader } from 'reactstrap';
+import Textarea from 'components/shared/Textarea';
+import { TextareaEvent } from 'components/shared/Textarea/Textarea';
+import Loading from 'components/shared/loading/Loading';
+
+interface Props {
+  contactUser: (input: ContactUserInput) => Promise<EmailResponse>;
+  booking: Booking;
+  handleModalAction: () => void;
```

Comment:
As above, I'd expect this to be named `onModalAction`

Reply:
👍 done